iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalCatalog.java

```diff
@@ -9,12 +10,17 @@
 import com.linkedin.openhouse.internal.catalog.model.HouseTablePrimaryKey;
 import com.linkedin.openhouse.internal.catalog.repository.HouseTableRepository;
 import com.linkedin.openhouse.internal.catalog.repository.exception.HouseTableRepositoryException;
+import com.linkedin.openhouse.internal.catalog.toggle.annotation.IcebergFeatureGate;
```

Comment:
We should move the classes out of `iceberg` module, they are generic enough to apply to any format. Also, storage classes wouldn't be able to use it easily if they are part of internalcatalog module.

Reply:
got it, addresssed.